pkgs/build-support/vm/default.nix

```
@@ -258,7 +257,7 @@ rec {
     # Write the command to start the VM to a file so that the user can
     # debug inside the VM if the build fails (when Nix is called with
     # the -K option to preserve the temporary build directory).
-    cat > ./run-vm <<EOF
+    ${coreutils}/bin/cat > ./run-vm <<EOF
```

Not sure what splicing is in this file, but intuitivly I would say this should be:

```suggestion
    ${lib.getBin buildPackages.coreutils}/bin/cat > ./run-vm <<EOF
```

---

But looks like this overall doesn't cross-compile well.